pjsip/include/pjsua2/call.hpp

```diff
@@ -562,6 +562,11 @@ struct CallInfo
  */
 struct StreamInfo
 {
+    /**
+     * Media stream instance, either audio or video.
+     */
+    MediaStream		stream;
+
```

Comment:
how about apps using SWIG such as Android, there seem to be no API to modify the codec param?

Reply:
Yes, because there's currently no API in the pjsua level as well (currently, the PR only provides access to the media stream and user will call PJMEDIA API directly).

So do we want to add pjsua&pjsua2 API as well? Perhaps something like `pjsua_call_aud_stream_modify_codec_param(call_id, med_idx, codec_param)`?

Reply:
Yes.

Btw, IMO direct access to pjmedia stream in pjsua/pjsua2 is better avoided for now, as there seem to be no urgent need of it for now, even it potentially introduces problems, e.g: SDP renego may recreate streams and there can be race cond or access-after-free issues when app accessing the stream without lock/ref-count protection.